platform-coop-toolkit/coop-map-directory-index
# feat: add worker distribution (resolves #74)

## Description

Adds worker distribution field to organizations.

## Steps to test

1. Check out this branch.
2. Run `./manage.py migrate mdi`.
3. Edit an organization via the admin user interface.

**Expected behavior:** A field for Worker distribution is present, with options for:

- Co-located
- Regionally distributed
- Nationally distributed
- Internationally distributed

## Additional information

Not applicable.

## Related issues

Resolves #74.
